core/src/test/java/kafka/test/api/ShareConsumerTest.java

```diff
@@ -1974,7 +1974,7 @@ private void warmup() throws InterruptedException {
         }
     }
 
-    private void maybeVerifyShareGroupStateTopicRecordCount(String persister, int messageCount) {
+    private void maybeVerifyShareGroupStateTopicRecordCount(String persister) {
```

Comment:
The method name now seems redundant. I suggest `verifyShareGroupStateTopicRecordsProduced`.